samples/cloud-client/snippets/restrict_api_key_api.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START auth_cloud_restrict_api_key_api]
+
+from google.cloud import api_keys_v2
+from google.cloud.api_keys_v2 import Key
+
+
+def restrict_api_key_api(project_id: str, location: str, key_id: str) -> Key:
+    """
+    Restrict an API key. Restrictions specify which APIs can be called using the API key.
+
+    // TODO(Developer): Replace the variables before running the sample.
```

Comment:
Global comment-- we tell them to replace the variables, but I don't see any variables, I just see sample values. Might it be easier for them to determine what they needed to replace if we listed them out, and used some kind of variable? What is the convention for this?

Reply:
Hi Pia, the TODO is to replace the values of the variables in the method.
I'll leave this comment open to get the input of the sample reviewers group.